Source/JavaScriptCore/b3/air/AirEmitShuffle.cpp

```diff
@@ -359,6 +359,8 @@ Vector<Inst> emitShuffle(
         result.append(Inst(move, origin, pair.src(), pair.dst()));
     };
 
+    bool freezeScratches = false;
```

Comment:
Instead of having a bool variable, can we pass a bool variable to `handleShift` to make it explicit from the caller?